### PR TITLE
[A11y]Fixing a missed horizontal scroll bar case

### PR DESCRIPTION
**Pull request by ryuyu**
Fixing https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1188115 being reopened.
image

image

### CODE REVIEW

**Comment by loic-sharma** [comment]
Do we need to update these snippets as well?

https://github.com/NuGet/NuGetGallery/blob/41db291887bb0a08ab7e7c086624df0331c3079c/src/NuGetGallery/Views/Users/_UserAccountChangeExternalCredential.cshtml#L72-L80

https://github.com/NuGet/NuGetGallery/blob/41db291887bb0a08ab7e7c086624df0331c3079c/src/NuGetGallery/Views/Users/_UserAccountChangeExternalCredential.cshtml#L84-L89

File: src/NuGetGallery/Views/Users/_UserAccountChangeExternalCredential.cshtml [hunk 1]
```diff
@@ -30,10 +30,10 @@
                 @Html.AntiForgeryToken()
 
                 <div class="login-account-row">
-                    <span>
+                    <span class="col-sm-9"  style="padding: 0;">
```
Comment (loic-sharma) [inline src/NuGetGallery/Views/Users/_UserAccountChangeExternalCredential.cshtml:33]: What's with the `padding: 0`?  Do these spans need to be wrapped by a `<div class="row"> ... </div>` or something?

Comment (ryuyu) [reply src/NuGetGallery/Views/Users/_UserAccountChangeExternalCredential.cshtml:33]: I tried that and it behaved very strangely. Currently, overriding this for this line was the most robust solution in practice in terms of reflow. btn-block appears to make the button behave oddly for reflowing.

**Comment by ryuyu** [comment]
Good catch on those snippets @loic-sharma 
Updated.

File: src/NuGetGallery/Views/Users/_UserAccountChangeExternalCredential.cshtml (hunk 1 above)
Comment (lyndaidaii) [reply src/NuGetGallery/Views/Users/_UserAccountChangeExternalCredential.cshtml:33]: I am aslo not sure` padding: 0` here, I tried build your changes locally, it appears display improperly
Capture

Comment (ryuyu) [reply src/NuGetGallery/Views/Users/_UserAccountChangeExternalCredential.cshtml:33]: Good eye @lyndaidaii ! I noticed when testing on dev that I forgot to push a CSS change that needs to accompany this. It is now pushed.

**Review by lyndaidaii — APPROVED** [review]

**Pull request merged**